sdk/lambdatesthyperexecute/arm-lambdatesthyperexecute/review/arm-lambdatesthyperexecute.api.md

```diff
@@ -0,0 +1,324 @@
+export interface OrganizationsGetOptionalParams extends OperationOptions {
+}
+
+// @public
+export interface OrganizationsListByResourceGroupOptionalParams extends OperationOptions {
+}
+
+// @public
+export interface OrganizationsListBySubscriptionOptionalParams extends OperationOptions {
+}
+
+// @public
+export interface OrganizationsOperations {
+    createOrUpdate: (resourceGroupName: string, organizationname: string, resource: OrganizationResource, options?: OrganizationsCreateOrUpdateOptionalParams) => PollerLike<OperationState<OrganizationResource>, OrganizationResource>;
+    delete: (resourceGroupName: string, organizationname: string, options?: OrganizationsDeleteOptionalParams) => PollerLike<OperationState<void>, void>;
+    get: (resourceGroupName: string, organizationname: string, options?: OrganizationsGetOptionalParams) => Promise<OrganizationResource>;
+    listByResourceGroup: (resourceGroupName: string, options?: OrganizationsListByResourceGroupOptionalParams) => PagedAsyncIterableIterator<OrganizationResource>;
+    listBySubscription: (options?: OrganizationsListBySubscriptionOptionalParams) => PagedAsyncIterableIterator<OrganizationResource>;
+    update: (resourceGroupName: string, organizationname: string, properties: OrganizationResourceUpdate, options?: OrganizationsUpdateOptionalParams) => Promise<OrganizationResource>;
+}
+
+// @public
+export interface OrganizationsUpdateOptionalParams extends OperationOptions {
+}
+
+// @public
+export type Origin = string;
+
+// @public
+export interface PagedAsyncIterableIterator<TElement, TPage = TElement[], TPageSettings extends PageSettings = PageSettings> {
+    [Symbol.asyncIterator](): PagedAsyncIterableIterator<TElement, TPage, TPageSettings>;
+    byPage: (settings?: TPageSettings) => AsyncIterableIterator<ContinuablePage<TElement, TPage>>;
+    next(): Promise<IteratorResult<TElement>>;
+}
+
+// @public
+export interface PageSettings {
+    continuationToken?: string;
+}
+
+// @public
+export interface PartnerProperties {
+    licensesSubscribed: number;
+}
+
+// @public
+export interface Resource {
+    readonly id?: string;
+    readonly name?: string;
+    readonly systemData?: SystemData;
+    readonly type?: string;
+}
+
+// @public
+export type ResourceProvisioningState = string;
+
+// @public
+export function restorePoller<TResponse extends PathUncheckedResponse, TResult>(client: HyperExecuteClient, serializedState: string, sourceOperation: (...args: any[]) => PollerLike<OperationState<TResult>, TResult>, options?: RestorePollerOptions<TResult>): PollerLike<OperationState<TResult>, TResult>;
+
+// @public (undocumented)
+export interface RestorePollerOptions<TResult, TResponse extends PathUncheckedResponse = PathUncheckedResponse> extends OperationOptions {
+    abortSignal?: AbortSignalLike;
+    processResponseBody?: (result: TResponse) => Promise<TResult>;
+    updateIntervalInMs?: number;
+}
+
+// @public
+export interface SingleSignOnPropertiesV2 {
```

Comment:
V2? Is this expected?

Reply:
@aggarwalsw The name is quite wired, do you know why we add the `V2` suffix here?

I may approve this pr since this is preview version but I think we need to refine with better one before GA.